web/api/v1/api.go

```diff
@@ -853,11 +854,18 @@ func (api *API) remoteRead(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	if req.ResponseType != prompb.ReadRequest_SAMPLES {
+	switch req.ResponseType {
+	case prompb.ReadRequest_SAMPLES: // Also default.
+		api.sampledRemoteRead(ctx, w, req)
+	case prompb.ReadRequest_STREAMED_XOR_CHUNKS:
+		api.streamedChunkedRemoteRead(ctx, w, req)
+	default:
 		http.Error(w, fmt.Sprintf("%s response type is not implemented", req.ResponseType.String()), http.StatusNotImplemented)
```

Comment:
This should fallback to SAMPLES

Reply:
hm.. Not sure about it, but if we move to `accepted_response_types` field instead, the logic will be clearer here, right? See https://github.com/prometheus/prometheus/pull/5701/files#r296731573

Reply:
Not sure, maybe for compatiblity falling back to sampled might make sense here